### ISSUE TITLE
[enhancement] Stopping the chat

### ISSUE THREAD

**Issue by rotello** [brianpetro/obsidian-smart-connections#60]
We all know how verbose chatGPT is. sometime i would like to stop it... to save GPU, time and money. the web version has a button to stop its flow of text. maybe the API let us do the same...

**Comment by smartguy1196** [comment]
@brianpetro Did you post a screenshot of the feature? I know that you normally do, but I can't seem to find this one.

**Comment by brianpetro** [comment]
> I imagine the next feature announcement will be pretty large

Yeah, I hope so. It is time-consuming. I think early next week would be the earliest. Plus, I still need some time to digest this [ChatGPT Plugins update](https://openai.com/blog/chatgpt-plugins) and what that means for the road map,

**Comment by smartguy1196** [comment]
# Here's a screenshot of the new button
@brianpetro is the button supposed to be an empty box, or is that some kind of rendering error?

![image](https://user-images.githubusercontent.com/12621101/227332618-8da61387-2b35-4357-901e-627a6c4f17f3.png)

**Comment by brianpetro** [comment]
The square is correct. It's like a stop button on a media player. Same as ChatGPT.

**Comment by smartguy1196** [comment]
Oh I get it. It looks more like an Obsidian Checkbox, so I was confused by it.

**Comment by brianpetro** [comment]
> Oh I get it. It looks more like an Obsidian Checkbox, so I was confused by it.

Yeah, I can see how it looks like that.

The ChatGPT stop icon never made much sense to me, either. But, I decided to use that UIUX as the basis for design decisions for the chat in Smart Connections, so I don't plan on changing it in the near term.

**Comment by smartguy1196** [comment]
Perhaps make it a solid square stop button (Similar to the stop button in DevTools) to avoid the confusion?

**Comment by brianpetro** [comment]
Maybe in the future, right now, I'm just sticking to parity with ChatGPT.
